packages/r/reveal.js.json

```
@@ -16,14 +16,8 @@
       {
         "basePath": "",
         "files": [
-          "css/**/*.css",
-          "js/*.js",
-          "lib/**/*.js",
-          "lib/**/*.css",
-          "lib/**/*.eot",
-          "lib/**/*.ttf",
-          "lib/**/*.woff",
-          "plugin/**/*.js"
+          "dist/**",
+          "plugin/**"
```

```suggestion
          "plugin/**/*.@(js|css)"
```
As above, open globstar is bad, let's restrict it to what we need :)